### DISCUSSION TITLE
Can´t get Alpine.data to work.

### DISCUSSION THREAD

**Discussion by zzseba78** [alpinejs/alpine#1484] [7. Bugs (archived)]
Can´t get Alpine.data to work.

I created a very simple example on codepen because i can´t get to work Alpine v3.

https://codepen.io/zzseba78/pen/YzZdzRr

It uses cdn and a simple example from docs, what i´m missing??

**Comment by SimoTod** [comment]
Your cdn script should be deferred otherwise Alpine initialises before you register the dropdown.

**Reply by zzseba78** [reply]
OMG, i was going crazy with this. One question, is ready for production or still in dev? Thanks!

**Reply by SimoTod** [reply]
I think it's in beta and not ready for production but I wasn't involved with v3 so it's just my personal take. For codepen, since it doesn't allow you to defer script easily, you can start it manually, see https://codepen.io/SimoTod/pen/eYvbmdz

**Reply by zzseba78** [reply]
Yes i defer it and worked, it will always require `defer` on it, even if i load the script at the bottom before closing body tag?

**Reply by SimoTod** [reply]
You just need to make sure you register your listeners before Alpine.start() which happens to be when your browser evaluates the cdn link. If the cdn link is after your custom js, it will work without defer
